.github/workflows/docker_images.yml

```diff
@@ -311,7 +311,7 @@ jobs:
           file: ./docker/build/cudaq.dev.Dockerfile
           build-args: |
             base_image=${{ steps.prereqs.outputs.base_image }}
-            install="CMAKE_BUILD_TYPE=Debug"
+            install="CMAKE_BUILD_TYPE=Debug CUDAFLAGS=-allow-unsupported-compiler"
```

Comment:
This looks sort of ominous. Is this going to be an on-going maintenance issue?

Reply:
This is what we have been doing so far; gcc12 is not officially supported by CUDA. Yes, it is ominous. The flag just wasn't explicitly given because of a hack that bypassed the cmake checks that would have complained about this.